### ISSUE TITLE
Invalid triples in 'identity document'

### ISSUE THREAD

**Issue by ubbc** [edmcouncil/fibo#930]
[https://spec.edmcouncil.org/fibo/ontology/FND/AgentsAndPeople/People/IdentityDocument](https://spec.edmcouncil.org/fibo/ontology/FND/AgentsAndPeople/People/IdentityDocument)

The IS-A restrictions include `verifies address max 1 physical address`

In both the Production and Development releases, the restriction is encoded using the triples:
```
[
    a owl:Restriction ;
    owl:onProperty fibo-fnd-aap-ppl:verifiesAddress ;
    owl:onClass fibo-fnd-plc-adr:PhysicalAddress ;
    owl:maxCardinality "1"^^xsd:nonNegativeInteger ;
] 
```
`owl:maxCardinality` should be replaced with `owl:maxQualifiedCardinality` to express a  qualified restriction.

**Comment by dallemang** [comment]
Good catch, Graham; not only should we fix this, but we should also make sure that we can automatically catch these in the future.

**Comment by ElisaKendall** [comment]
I agree - hygiene test?  This has been there a really long time.  Will try to address this and the others you've raised today.

**Comment by ubbc** [comment]
You are welcome.
It is actually surprisingly difficult to find all the things that can go wrong as most tools are surprisingly "permissive" when parsing. I've been working on a tool-set that implements the strict parsing rules set out in [Mapping to Structural Spec](https://www.w3.org/TR/owl2-mapping-to-rdf/#Mapping_from_RDF_Graphs_to_the_Structural_Specification) following the Canonical steps.
It then relaxes the rules (for example requiring declarations) and the then reports all the things it had to do (or couldn't parse). 

They are not quite production-ready yet but as a test I was able to load the entire FIBO set of Production ontologies - these are the things it through out.

I found another issue with BusinessDayAdjustmentCode in the Production release that relates to an ontology not being specifically imported before being used. However, I noticed this is being significantly refactored in the next release so I haven't raised it until I track it down further.  I also found some others in the Development version but I need to look more closely at it when i have a moment.
